<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deck</class>
 <widget class="QWidget" name="Deck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="add_card_btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
background-color: rgb(50, 54, 63);
color: white; 
font:bold;
border: 0px;
}

QToolButton:hover {
	background-color: rgba(0, 127, 177, 0.6);
}</string>
       </property>
       <property name="text">
        <string>Add Card</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="quiz_btn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolButton {
background-color: rgb(50, 54, 63);
color: white; 
font:bold;
border: 0px;
}

QToolButton:hover {
	background-color: rgba(0, 127, 177, 0.6);
}

QToolButton:checked {
	background-color: rgb(0, 127, 177);
}</string>
       </property>
       <property name="text">
        <string>Quiz</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="total">
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	color: white;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stacked_deck">
     <widget class="QWidget" name="card_view">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="back_to_deck_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>8</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QToolButton {
background-color: rgb(50, 54, 63);
color: white; 
font:bold;
border: 0px;
}

QToolButton:hover {
	background-color: rgba(0, 127, 177, 0.6);
}</string>
         </property>
         <property name="text">
          <string>Back to Deck</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="card_area"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="prev_btn">
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgb(30, 33, 36);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(0, 127, 177, 0.6);
}

QPushButton:checked {
	background-color: rgb(0, 127, 177);
}</string>
           </property>
           <property name="text">
            <string>Prev</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="next_btn">
           <property name="font">
            <font>
             <family>Trebuchet MS</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
background-color: rgb(30, 33, 36);
color: white; 
font:bold;
padding: 8px 12px 8px 12px;
border: 0px;
}

QPushButton:hover {
	background-color: rgba(0, 127, 177, 0.6);
}

QPushButton:checked {
	background-color: rgb(0, 127, 177);
}</string>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="deck_view">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="card_list_wrapper">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>549</width>
            <height>336</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="card_list"/>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>302</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
